### DISCUSSION TITLE
How do I make the seconds go up every second?

### DISCUSSION THREAD

**Discussion by Drogobo** [i3/i3#5014] [Q&A] [answered]
Right now, the seconds update every 5 seconds, so I get time like 05, 10, 15. I want it to give me 00, 01, 02 and so on. Basically, I want it to count by one and not five.

**Comment by Cippo95** [comment, marked as answer]
You want to change the interval in i3status.
https://i3wm.org/i3status/manpage.html#_configuration

**Reply by Drogobo** [reply]
thx girly
